akka-stream/src/main/scala/akka/stream/impl/Timers.scala

```diff
@@ -59,12 +59,12 @@ import akka.stream.stage._
 
         final override protected def onTimer(key: Any): Unit =
           if (!initialHasPassed)
-            failStage(new TimeoutException(s"The first element has not yet passed through in $timeout."))
+            failStage(new InitialTimeoutException(s"The first element has not yet passed through in $timeout."))
```

Comment:
`toString` of `FiniteDuration` isn't always very readable. Use `timeout.toCoarsest`